gensim/test/test_fasttext.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Author - Prakhar Pratyush (er.prakhar2b@gmail.com)
+
+import logging
+import unittest
+import os
+from gensim import utils
+
+from gensim.models.fasttext import FastText
+
+module_path = os.path.dirname(__file__)  # needed because sample data files are located in the same folder
+datapath = lambda fname: os.path.join(module_path, 'test_data', fname)
+
+
+class LeeCorpus(object):
+    def __iter__(self):
+        with open(datapath('lee_background.cor')) as f:
+            for line in f:
+                yield utils.simple_preprocess(line)
+
+
+class TestNativeFastText(unittest.TestCase):
+
+    def testTraining(self):
+        """ Test training with subwords for both skipgram and cbow model"""
+        sentences = LeeCorpus()
+        self.assertTrue(FastText(sentences, min_count=100, size=100, workers=3))
+        self.assertTrue(FastText(sentences, sg=1, min_count=100, size=100, workers=3))
```

Comment:
Model sanity tests similar to word2vec are required here too to ensure the "correct" models are learnt, as well as a comparison to the original FastText vectors.